README.md

```diff
@@ -1 +1,34 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+
+> A simple webpage that displays a greeting
+
+## Built With
+
+- HTML
+- CSS
+
```

Comment:
- [ ] [REQUIRED] Kindly include other headers as it is in the template. You can add the `Getting Started` header and tell us how we can clone and use your code.